starknet_py/net/udc_deployer/deployer.py

```diff
@@ -176,3 +183,15 @@ def _get_random_salt() -> int:
 _deployer_serializer = serializer_for_function(
     _deployer_abi.functions["deployContract"]
 )
+
+
+def _is_list_of_ints_or_strings(data: Union[List, dict]) -> bool:
+    """
+    Checks if the given data is a list containing only strings or integers.
+
+    :param data: Constructor args (calldata) of the contract to be deployed.
+    :return: True if the data is a list of strings or integers, False otherwise.
+    """
+    if isinstance(data, list):
+        return all(isinstance(x, (int, str)) for x in data)
+    return False
```

Comment:
Is it desired for this function to return `True` is the list is empty?

Reply:
Yes, I've tested it